bin/gen-crons.py

```diff
@@ -23,6 +23,8 @@ def main():
                             'Only define for cron.d style crontabs.'))
     parser.add_option('-p', '--python', default='python2.6',
                       help='Python interpreter to use.')
+    parser.add_option('-n', '--nologdir', action='store_true',
```

Comment:
What if we just remove the default on line 10, and add the CRON_LOG_DIR to the real environment and leave it out in our testing environment?